spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe FoodFacade do
+      it 'returns Food data', :vcr do
+        food_results = FoodFacade.find_food("sweet potatoes")
+
+        expect(food_results.first.gtin_upc).to eq("8901020020844")
```

Comment:
This can be a bit tricky in case the first food changes in the API and VCR re-records -- it might change. For the purpose of the assessment this is totally fine/reasonable but in a project I'd suggest testing exact values in the POROs instead where you can easily stub the data and make assertions about attributes getting populated with the right thing